src/software/jetson_nano/services/motor.cpp

```diff
@@ -607,7 +611,7 @@ void MotorService::spiTransfer(int fd, uint8_t const* tx, uint8_t const* rx, uns
 {
     int ret;
 
-    struct spi_ioc_transfer tr[1];
+    struct spi_ioc_transfer tr[2];
```

Comment:
Curious if this change is necessary? Since it seems like the rest of this method is unchanged.

Reply:
good catch that was leftover from a refactor that didn't go as planned

Reply:
fixed